doc/.static/version_switch.js

```diff
@@ -53,7 +53,7 @@
 
     vswitch.push('<dl>');
     vswitch.push('<dt>On GitHub</dt>');
-    var git_master = "https://github.com/pymc-devs/aesara"
+    var git_master = "https://github.com/aesara-devs/aesara"
     vswitch.push('<dd><a href=\"', git_master + '\">', 'Fork me', '</a></dd>');
```

Comment:
Is that a reference to the old `master` branch?

Reply:
Not sure; it uses `master` in the JS variable name, but doesn't/didn't seem to refer to a specific branch.

Reply:
Oh it's just a variable name, I didn't see that. Then it's nothing to worry about.